css/custom.scss

```diff
@@ -59,7 +59,7 @@ $link-active: $link-hover;
   background-position: center center;
   background-repeat: no-repeat;
   background-size: cover;
-  height: 450px;
+  height: auto;
```

Comment:
This change belongs to your other PR I think

Reply:
yes it would be great

Reply:
Done. Thank you. 🙌🏼